Frameworks/include/MockClass.h

```diff
@@ -21,32 +21,49 @@
 #define __alignof alignof
 #endif
 
-#include "ParameterTypes.h"
+#define BUILD_VARIABLE_NAME_INT(x, y) x##y
+#define BUILD_VARIABLE_NAME(x, y) BUILD_VARIABLE_NAME_INT(x, y)
+
+#define MEMBER_PARAMETER_TYPES_CONST_false
+#define MEMBER_PARAMETER_TYPES_CONST_true const
+
 #include "Windows.h"
 
+#include <tuple>
+
 // Copying these macros from WexTestClass.h, but not including
```

Comment:
Based on this comment, the following macros could simply be destroyed? Or does this exist for cross-platform support?